OMX-27-firmware/OMX-27-firmware.ino

```diff
@@ -1626,7 +1822,7 @@ void loop() {
 					case MODE_S2:
 						if (!patternSettings[playingPattern].solo){
 							if (keyState[1] && keyState[2]) {	
-//								seqPages = true;
+								seqPages = true;
 								
 							} else if (!keyState[1] && !keyState[2]) {  // SKIP LONG PRESS IF FUNC KEYS ARE ALREDY HELD
```

Comment:
yeah `keyState` is a bool flag for if that key is being held. 1 and 2 are the Function keys.

I think this would change anyway if we revamp the key press/hold routines.